### ISSUE TITLE
`make install` fails on Windows

### ISSUE THREAD

**Issue by lstor** [twolfson/sexy-bash-prompt#45]
Using ConEmu and Git Bash (MsysGit), `make install` fails on copying `.bash_prompt`. I'm not sure exactly what caused it, but simply running `touch ~/.bash_prompt` and then re-running `make` fixed the problem.

```
$ make install
# Copying .bash_prompt to ~/.bash_prompt
cp -f .bash_prompt ~/.bash_prompt
cp: cannot create regular file `~/.bash_prompt': No such file or directory
Makefile:6: recipe for target 'install' failed
mingw32-make: *** [install] Error 1
$ touch ~/.bash_prompt
$ make install
# Copying .bash_prompt to ~/.bash_prompt
cp -f .bash_prompt ~/.bash_prompt
./install.bash
# Adding ~/.bash_prompt to ~/.bashrc
# twolfson/sexy-bash-prompt installation complete!
exit 0
```

**Comment by twolfson** [comment]
Interesting, maybe `-f` has a different behavior inside of `msysgit`. I will take a look at this by the end of the weekend.

**Comment by lstor** [comment]
`msysgit` uses GNU `cp`, so there shouldn't be a difference -- it's `--force`. I have no idea right now why it failed to copy just because the target file wasn't present.

**Comment by twolfson** [comment]
This has been fixed/released in `0.26.0`. Thanks for the report and all your help with the issue =)

**Comment by lstor** [comment]
Verified fixed now :) I tried to install it again, and it worked directly with the one-liner this time. Thanks!
